vendor/bat-native-ads/src/bat/ads/internal/ad_events/ad_notification_event_clicked.cc

```diff
@@ -34,6 +34,8 @@ void AdNotificationEventClicked::Trigger(
   ads_->get_ad_notifications()->Remove(ad_notification.uuid,
       /* should dismiss */ true);
 
+  ads_->set_last_shown_ad_notification(ad_notification);

```

Comment:
nit: If I understand correctly the corresponding variable in `AdsImpl` is only set on click now. So maybe rename to `set_last_clicked_ad_notification()`  and corresponding to that in `AdsImpl` rename `last_shown_ad_notification_` to `last_clicked_ad_notification_`.

Reply:
Renaming to `last_shown_ad_` in an upcoming PR to support different ad types